cmd/libs/go2idl/args/args.go

```diff
@@ -71,11 +74,14 @@ type GeneratorArgs struct {
 }
 
 func (g *GeneratorArgs) AddFlags(fs *pflag.FlagSet) {
+	//FIXME: can we simplify operation to one input file, one output?
```

Comment:
Is this a case of we could start with a single file and process all deps of that file, and eventually emit a single file?  Can you describe the algorithm for protobuf a bit more?